helpers/extension/driver.js

```diff
@@ -150,20 +157,54 @@ class ExtensionProtocol extends ChromeProtocol {
     });
   }
 
-  // Stubs to bypass a page reload for now.
-  /* eslint-disable no-unused-vars */
   off(eventName, cb) {
-    return undefined;
+    if (typeof this._listeners[eventName] === 'undefined') {
+      return;
+    }
+
+    const callbackIndex = this._listeners[eventName].indexOf(cb);
+    if (callbackIndex === -1) {
+      return;
+    }
+
+    this._listeners[eventName].splice(callbackIndex, 1);
   }
 
-  gotoURL(url, waitForLoad) {
+  gotoURL(url, waitForLoaded, reloadPageAndRunAllTests) {
+    if (reloadPageAndRunAllTests) {
```

Comment:
inverting the guard clause here if (!....) {return Promise.resolve()} would let us outdent all of this code :sparkles: